CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# 0.2.0
+
+Changed location of config file
+
+now following the OS standard
+
+    // Linux:   /home/alice/.config/pndev/pndev.toml
+    // macOS:   /Users/Alice/Library/Preferences/rs.pndev.pndev/pndev.toml
```

Comment:
Why not `/Users/Alice/.config/pndev/pndev.toml` on macOS?

Reply:
Turns out that that's not the standard apple wants people to follow and that applications are supposed to use the directory structure above instead.
This is the behavior implemented by default in the confy crate.

We can change it of course but then we either have the bug that this pr refers to or I re implement the functionality from the configuration crate I'm using.

I can probably do the latter without much effort tbh

Was just doing the easiest thing and following apple's bidding.

I'm not a fan of having it different on OSX but you edit the file once anyway so I'm ok with it overall

Reply:
also @typeoneerror see the link in the PR description for context